README.md

```diff
@@ -0,0 +1,21 @@
+# foo-config-loader
+
+A config loader for the Foo project.
+
+
+# Project tasks
+
+## In order to setup a local env run...:
+
+    virtualenv create -p python3.6 venv/
```

Comment:
Does this project works only with `python3.6` ? 
How about giving a way of installing `python3.6` first?